### DISCUSSION TITLE
Woodworkin apron

### DISCUSSION THREAD

**Discussion by Whaitte** [sunflower-land/sunflower-land#1389] [Art]
With the expansion of lands coming and along with it the halv, me and Lucas Berserker made this clothing planning for Bumpkin, which has as proposal to give some benefit when buying tools in the blacksmith.

We made these two models:

![Untitled 08-30-2022 10-41-16 (4).png](https://user-images.githubusercontent.com/104186329/188644510-90fdc7ae-2f9c-4d0c-8ab9-49418a46599e.png)![Untitled 08-30-2022 10-41-16 (3).png](https://user-images.githubusercontent.com/104186329/188644531-6240ab60-f03e-4022-97f9-b2318d7c955d.png)
![Untitled 08-30-2022 10-41-16 (2).png](https://user-images.githubusercontent.com/104186329/188644626-c8300a74-8717-4660-8579-ce3813eeafa6.png)![Untitled 08-30-2022 10-41-16.png](https://user-images.githubusercontent.com/104186329/188644641-16c830fe-02c4-4f85-a513-35aec90d870a.png)

We have two suggestions for the buff:

**1st** - Discount of 10% - 20% on all tools, examples with 20%:
• Official Value of Tools:
Axe:         1SFL
Pickaxe:     1SFL + 2 Wood
Stone Pickaxe:     2 SFL + 3 Wood + 3 Stone
Iron Pickaxe:     5 SFL + 5 Wood + 3 Iron

• Value using Woodworking Apron
Axe:         0.8 SFL
Pickaxe:     0.8 SFL + 1.6 Wood
Stone Pickaxe:     1.6 SFL + 2.4 Wood + 2.4 Stone
Iron Pickaxe:     4 SFL + 4 Wood + 2.4 Iron

**2nd** - The second is simpler, giving 10 to 20% more tools when buying, so with each purchase with blacksmith, instead of taking only 1 tool, we would get 1.2 using Woodworking Apron (this with the 20% buff)

- Suggestion for Craft with 20% buff:       
Supply: 500  

50 Golds         
100 irons  
1000 Woods  
1000 Stones
100 SFL

- Suggestion for Craft with 10% buff:                                                                 Supply: 1000

25 Golds                                                                     
50 irons                                                              
1000 Woods                                                                 
500 Stones                                                                           
75 SFL

**Bumpkin examples:**
![189238238-4e58cd91-c955-459c-87e6-f9d1d52603df (2).png](https://user-images.githubusercontent.com/104186329/190144610-54541b54-be0b-433c-8e0e-7cf86b986f24.png)

![189238241-ef821e46-da4c-4e45-8aca-ec01adb5b25f.png](https://user-images.githubusercontent.com/104186329/190144642-fa6a0bf3-6c8b-4513-9a35-3eac1af0e3f8.png)

Farm ID:#919
Farm address:0xBBb40e3E4359c7af0254728ab52A4c39a14f5c08
Discord: whaitte#3465

Farm ID: #6238
Farm address:
0x7229DE9907F535522311d236Df187E52cEf77a15
Discord: Lucas Berserker#9158

**Comment by LucasAG95** [comment]
_**I'm really excited about our work**_

**Reply by denis3333** [reply]
It would be great to have one of these, I'm looking forward to it

**Comment by NanndoSnowzera** [comment]
I liked it, I think it's well balanced, if I change something it would be a small thing. very good, it will be nice to have this in the land expansion.

**Comment by Tk175C** [comment]
i love it and i want it

**Comment by xSubZero** [comment]
Hello everyone!

I really liked the proposal, I believe it will be pleasant for everyone, I really need an apron like that.

About the two arts, I prefer the second, with the pocket below. 

Congratulations on her idea and development 💙

**Comment by MAISAbr** [comment]
I loved the proposal I really liked the color 💙

**Comment by Alexcnder** [comment]
**I love the idea and the art but i think its a little cheap**

rn the costs:
10 axes 10SFL
10 Pickaxes 15SFL
10 Stone Pickaxes 42SFL
5 Iron Pickaxes 55SFL
 plus  20 SFL
 thats in total 142 SFL
 
 And you would save
 on the pickaxes 0,2 SFL + 0,4 Wood = 0,2SFL + 0,4*0,1SFL = 0,24 SFL
 stone pickaxes 0,4 SFL + 0,6 Wood + 0,6 Stone = 0,4SFL + 0,6*0,1SFL + 0,6*0,25SFL = 0,61 SFL
 Iron pickaxe 1SFL + 1 Wood + 0,6 Iron = 1SFL + 1*0,1SFL + 0,6*1,2SFL = 1,82 SFL
 
 you can use pickaxes 9 times a day so 2,14 SFL
 you can use 3 stone pickaxes on average a day so 1,83 SFL
 and the iron pickaxe you can use once a day so 1,82 SFL
 
 so in total you would save 5,79 SFL a day
 so it would have a ROI of 25 days what seems a bit to good I would add atleast 5 More Iron Pickaxes to the cost that would make the Roi become a little more balanced
 
 (for the first suggestion)

 
 But in general this is a wonderful apron

**Reply by LucasAG95** [reply]
Thanks for the suggestion, I believe you are right, let's talk and modify, really the cost is much lower, it would be for all resources, in a way it has to cost more SFL
I believe the value of 300 SFL would be close to ideal (sum of resources with SFL)

**Reply by Alexcnder** [reply]
> Thanks for the suggestion, I believe you are right, let's talk and modify, really the cost is much lower, it would be for all resources, in a way it has to cost more SFL I believe the value of 300 SFL would be close to ideal (sum of resources with SFL)

yeah 300 SFL seems way better that like a good 50 days roi

**Reply by LucasAG95** [reply]
updated proposal

**Comment by aimlemonade** [comment]
I think the 1st buff is the better, waiting to get mine 👀

**Comment by Transvariando** [comment]
I really liked the art (the bottom one is the best)
Very strong idea, congratulations to the 2 for sure it will happen !!!

**Comment by Blasta-alasta** [comment]
Wicked idea, great design. Look forward to seeing this progress!

**Comment by future2back** [comment]
Supply 500? You recieve 1 for painting original one, dev team take 100 and people kill themselves for 399. Soon you sell more expensive on opensea, great idea bro! More boosts for big ones! Where do I sign?

**Comment by adamhannigan** [comment]
Great work @Whaitte 

Art looks on point 👌 This sounds like a nice boost for players to earn.

Would you be able to provide some examples of how this looks on different styles of Bumpkins, including the Goblin 😄

**Reply by Whaitte** [reply]
bumpkin ready❤️

**Reply by adamhannigan** [reply]
@Whaitte Looking good!

I would suggest trying to touch up the left arm a bit. The shirt is not quite aligned at the moment with the arm

**Reply by Whaitte** [reply]
@adamhannigan you have good eyes.

 I fixed it and tried to join the shadow of the arm with the one of the shirt, what do you think?
